### PR TITLE
Add configuration for ignoring enum members

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by EPashkin** [comment]
Maybe this need be added to SYS mode.

**Comment by sdroege** [comment]
Maybe, but I don't need / want it there for my use case :)

File: src/codegen/enums.rs [hunk 1]
```diff
@@ -109,7 +109,8 @@ fn generate_enum(env: &Env, w: &mut Write, enum_: &Enumeration, config: &GObject
     for member in &enum_.members {
         let member_config = config.members.matched(&member.name);
         let is_alias = member_config.iter().any(|m| m.alias);
-        if is_alias || vals.contains(&member.value) {
+        let ignore = member_config.iter().any(|m| m.ignore);
+        if is_alias || vals.contains(&member.value) || ignore {
```
Comment (EPashkin) [inline src/codegen/enums.rs]: Better move  `|| vals.contains(&member.value)` to last

**Comment by EPashkin** [comment]
Then only small fix and I merge this.

**Comment by sdroege** [comment]
Done

**Comment by EPashkin** [comment]
@sdroege Thanks.

**Pull request merged**